### DISCUSSION TITLE
BMW X1 xdrive25e F48 2020

### DISCUSSION THREAD

**Discussion by carroarmato0** [bimmerconnected/bimmer_connected#192] [Fingerprint data contributions]
[status_0.txt](https://github.com/bimmerconnected/bimmer_connected/files/5033378/status_0.txt)
[vehicles_0.txt](https://github.com/bimmerconnected/bimmer_connected/files/5033379/vehicles_0.txt)

**Comment by gerard33** [comment]
Thanks for sharing the fingerprint data.
